dali/driver/uart.py

```diff
@@ -0,0 +1,109 @@
+
+"""
+UART communication driver for DALI
+"""
+
+import logging
+import serial
+import struct
+from dali.command import Command
+from dali.interface import DriverInterface, ParameterError
+
+
+class DaliUART(DriverInterface):
```

Comment:
Ok. Description is [here](https://github.com/caiwan/python-dali/blob/master/dali/driver/uart.py#L46)